### PR TITLE
Remove additional_build_steps_*.sh support

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
There is no need to have it as preparation steps could be fit into prepare_*.sh

### CODE REVIEW

**Comment by AlexanderYastrebov** [comment]
@hundredwatt @gunnarmorling @thomaswue

File: evaluate2.sh [hunk 1]
```diff
@@ -236,14 +229,14 @@ for fork in "$@"; do
...
   # Hard-coding the note message for now
   notes=""
-  if [ -f "./additional_build_steps_$fork.sh" ]; then
-    notes="GraalVM native binary"
+  if [ -f "./prepare_$fork.sh" ]; then
+    java_version=$(grep -F "sdk use java" ./prepare_$fork.sh | cut -d' ' -f4)
+
+    if grep -F "native-image" -q ./prepare_$fork.sh ; then
+      notes="GraalVM native binary"
+    fi
```
Comment (hundredwatt) [inline evaluate2.sh:239]: Looks good to me!

File: prepare_thomaswue.sh [hunk 2]
```diff
@@ -17,3 +17,9 @@
 
 source "$HOME/.sdkman/bin/sdkman-init.sh"
 sdk use java 21.0.1-graal 1>&2
+
+# ./mvnw clean verify removes target/ and will re-trigger native image creation.
+if [ ! -f target/CalculateAverage_thomaswue_image ]; then
```
Comment (hundredwatt) [inline prepare_thomaswue.sh:22]: Not sure we want this, what if there was a code change since the last run?

**Review by hundredwatt — COMMENTED** [review]
Change looks good to me, I don't see any downside

File: prepare_thomaswue.sh (hunk 2 above)
Comment (AlexanderYastrebov) [reply prepare_thomaswue.sh:22]: `# ./mvnw clean verify removes target/ and will re-trigger native image creation.`

Comment (hundredwatt) [reply prepare_thomaswue.sh:22]: Oh, didn't notice you moved the output to the target directory

**Comment by gunnarmorling** [comment]
Yeah, nice improvement. Great to see how the pieces come together. Thx @AlexanderYastrebov, and thx @hundredwatt for reviewing! Merging.

**Pull request merged**